packages/design-system/src/components/Icons/AlertIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import type * as React from 'react';
+import { t } from '../i18n';
+import { SvgIcon, IconCommonProps } from './SvgIcon';
+
+const defaultProps = {
+  className: '',
+  viewBox: '0 0 5 17',
+};
+
+export function AlertIcon(props: IconCommonProps): React.ReactElement {
+  const iconCssClasses = `ds-c-icon--alert ${props.className || ''}`;
+  return (
+    <SvgIcon title={t('icons.warning')} {...defaultProps} {...props} className={iconCssClasses}>
```

Comment:
What do you think about changing this to:
```suggestion
    <SvgIcon title={t('icons. alertCircle')} {...defaultProps} {...props} className={iconCssClasses}>
```
so that @jack-ryan-nava-pbc's [point](https://github.com/CMSgov/design-system/pull/3574/files#r2082466060) means no translations are added?

Reply:
Yep, happy to make that change! I wasn’t sure if we needed a dedicated space in the translation json for each of these icons, but using the `icons.alertCircle` property makes sense to me.

Reply:
Also, just to clarify—using `icons.alert` was the intention here. I must have added `icons.warning` by mistake.